src/lib/reducers/apps.js

```diff
@@ -28,6 +28,7 @@ export const setInfos = (appName, editor) => ({ type: SET_INFOS, appName, editor
 export const fetchApps = () => async dispatch => {
   try {
     const rawAppList = await stack.get.apps()
+    const comingSoonApps = await fetchComingSoonApps()
     const apps = rawAppList.filter(app => !EXCLUDES.includes(app.attributes.slug))
```

Comment:
Not related to this PR, but `EXCLUDES.includes` is a funny syntax :)

Reply:
Agree^^